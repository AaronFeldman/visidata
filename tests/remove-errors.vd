sheet	col	row	keystrokes	input	comment
			o	sample_data/surveys.csv	open file
surveys	record_id	0	=	year+"-"+month+"-"+day	create new column from Python expression, with column names as variables
surveys	year+"-"+month+"-"+day	0	^	date	edit name of current column
surveys	month	0	-		hide current column (to unhide go to Columns sheet and edit its width)
surveys	day	0	-		hide current column (to unhide go to Columns sheet and edit its width)
surveys	year	0	-		hide current column (to unhide go to Columns sheet and edit its width)
surveys	date	0	@		set type of current column to ISO8601 datetime
surveys	date	0	:	 	add new columns from regex split; # columns determined by example row at cursor
surveys	date_re1	0	-		hide current column (to unhide go to Columns sheet and edit its width)
surveys	date	0	-		hide current column (to unhide go to Columns sheet and edit its width)
surveys	date_re0	0	^	date	edit name of current column
surveys	hindfoot_length	0	#		set type of current column to int
surveys	weight	0	#		set type of current column to int
surveys	weight	0	I		open Describe Sheet
surveys_describe	errors	10	q		quit current sheet
surveys	weight	0	gd		delete all selected rows and move them to clipboard
surveys	species_id	0	!		pin current column to the left as a key column
surveys	hindfoot_length	0	+	mean	add aggregator to the current column
surveys	sex	0	W		Pivot the current column into a new sheet
