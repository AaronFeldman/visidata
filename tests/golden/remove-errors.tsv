species_id	Total_hindfoot_length_mean	M_hindfoot_length_mean	F_hindfoot_length_mean	_hindfoot_length_mean	Total_count
DM	36.10	36.35	35.81		536
DO	33.91	34.44	33.27		34
PF	14.95	15.00	14.90		57
OX	20.00	19.00	21.00		3
PP	21.35	21.77	20.92		26
PE	21.07	20.89	21.40		16
OT	19.41	19.55	19.36	18.00	52
OL	20.57	20.60	20.56	20.00	40
RM	17.00	17.50	16.50		4
DS	49.84	50.82	49.31		329
NL	32.43	32.46	32.40		32
PM	20.00	20.00	20.00		3
SS					1
SH	29.00	29.00			1
